node_tree.py

```diff
@@ -307,6 +478,46 @@ def draw_color(self, context, node):
         return self.nodule_color
 
 
+class SvLinkMatrix(bpy.types.Operator):
```

Comment:
most nodes are the same width, so a X offset could have a default of 200, which it uses unless the operator is explicitely passed a different X offset.